frontend/components/ui/navbar/index.js

```diff
@@ -1,16 +1,95 @@
-import Link from 'next/link'
+import Link from "next/link";
+import styled from "styled-components";
+import React, { useState } from "react";
+
+// import MenuIcon from "../../hamburgerMenu/index";
+// HamburgerMenu svg can be used in place of the current → being used
 
 const Navbar = () => {
+	const [menu, setMenu] = useState(false);
+	// toogleMenu sets the menu to true which allows for conditional rendoring on the Nav items
+	// on initial load there is currently a temporary "Catch Phrase", once the user hits the →
+	// it sets memu to true, which renders the navbar items
+	const toggleMenu = () => {
+		setMenu(!menu);
+	};
+
 	return (
-	  <nav className="navbar">
-		<Link  href="/"><a>Home</a></Link>
-		{/* Next.js uses Link to wrap an <a> tag which allows for users to navigate between pages - The code above will return the user to the home page located at localhost:3000/ */}
-		<Link  href="/"><a>Start a Vote</a></Link> 
-		{/* Temp href for the time being until pages are set up */}
-		<Link  href="/temp/temp-landing"><a>Temp Landing</a></Link>
-	  </nav>
+		<>
+			<Nav className="navbar">
+				<p style={{ display: !menu ? "flex" : "none" }}>
+					Where Democracy Meets
+				</p>
+				<div style={{ display: menu ? "flex" : "none" }}>
+					<Link href="/">
+						<a>Home</a>
+					</Link>
+					{/* Next.js uses Link to wrap an <a> tag which allows for users to navigate between pages - The code above will return the user to the home page located at localhost:3000/ */}
+					<Link href="/">
+						<a>Start a Vote</a>
+					</Link>
+					{/* Temp href for the time being until pages are set up */}
+					<Link href="/temp/temp-landing">
+						<a>Temp Landing</a>
+					</Link>
+				</div>
+				<Button
+					style={{
+						position: "absolute",
+						backgroundColor: "transparent",
+						border: "none",
+					}}
+					onClick={toggleMenu}
+				>
+					{!menu && <ButtonText>&#8594;</ButtonText>}
+					{/* <MenuIcon /> */}
+				</Button>
```

Comment:
Small note here, I think we can render the button conditionally rather than just the button text that way we don't have an invisible button on the page:
```
{!menu && (
          <Button
            style={{
              position: "absolute",
              backgroundColor: "transparent",
              border: "none",
            }}
            onClick={toggleMenu}
          >
            <ButtonText>&#8594;</ButtonText>
            {/* <MenuIcon /> */}
          </Button>
        )}
```